src/sentry/static/sentry/app/views/settings/organizationMembers/inviteMember/index.jsx

```diff
@@ -203,6 +203,26 @@ const InviteMember = createReactClass({
     });
   },
 
+  handleSelectAll() {
+    let {teams} = this.getOrganization();
+    let {selectedTeams} = this.state;
+    let allSelected = teams.length === selectedTeams.size;
+
+    if (allSelected) {
+      teams.map(team => {
```

Comment:
Can we just clear the set?
```suggestion
      selectedTeams.clear()
```